advocacy_docs/migrating/oracle/factors_to_consider.mdx

```diff
@@ -68,6 +69,20 @@ Migrating a legacy database to new database technology results in the need to up
 
 Also, when legacy databases are running on older specialized hardware and operating systems, a database migration might also require migrating the underlying host systems to modern platforms and operating systems.  For on-premises migrations, this can result in the need to change your operations and maintenance procedures and tools to work with the new server platforms and operating systems. For migrations to the cloud, some of the operations and maintenance activities will be handled by the cloud service provider but others will still be your responsibility. Likely, those activities will require different tools and procedures from the ones you're currently using.
 
+## Sizing and configuration considerations
+
+"How big a server will I need?" 
+
+This question comes up often when considering Oracle-to-Postgres migrations. The shortest answer is, "It depends," which isn't a good answer. A better one is, "Start with a 1:1 correspondence (that is, same number of cores, RAM, and so on), and go from there."
+
+Why is that?
+
+Postgres and Oracle are very different under the hood, with different SQL planning and different storage and access patterns. Even the version control and transaction isolation are implemented in a different way. That makes it difficult to set up a simple formula that covers all possible permutations of Oracle configurations and architectures. All databases respond differently to the workload being run.
+
+EDB has performed many benchmarks on different CPU/RAM configurations that show that Postgres and EDB Postgres Advanced Server are both slightly faster than Oracle on a TPC-C workload. These results might not apply to your applications, but thye offer a good starting point. 
```

Comment:
This will likely lead to additional questions. I recommend adding a statement like, 'Customers of EDB who have migrated to EPAS have found performance on par with Oracle when using similar hardware. In some instances, depending on their workload, they even outperform Oracle. All our customers who migrated aimed to at least match Oracle's performance and, if feasible, to exceed it either on comparable hardware or by adding resources as needed. We (EDB) also suggest conducting a benchmark using your specific workload to ensure the system meets performance expectations and to plan production accordingly.

Reply:
You are right. Let's keep our testing private. Removed the section, and replaced it with yours.